intel_extension_for_transformers/llm/runtime/graph/models/falcon/falcon.h

```diff
@@ -30,6 +30,8 @@ static const model_scratch falcon_mem_req(int n_layers) {
     // TODO(hengyu): add more variants besides 6B
     case 60:
       return {2 * 2048ull * MB, 2 * 2048ull * MB, 2 * 4096ull * MB};
+    case 80:
+      return {3 * 2048ull * MB, 3 * 2048ull * MB, 3 * 4096ull * MB};
```

Comment:
please make sure the size is fit not too large 
since  layer 60 is  2 * 2048, maybe layer 80 should be 2.7 ?

Reply:
layer 60 is for 40B, layer 80 is for 180 B. Width (param) and depth (layer) both need to be considered. The mem scratch way should be upgraded into auto-alloc arch in the future.